js/customer.js

```diff
@@ -0,0 +1,95 @@
+/*================= Form validation =================*/
+window.addEventListener('load', function () {
+    submitForm.addEventListener('click', function (event) {
+        event.preventDefault();
+        const name = document.getElementById('name');
+        const email = document.getElementById('email');
+        const phoneNumber = document.getElementById('phoneNumber');
+        function validEmail (email) {
+            return email.value.indexOf("@") !== 0 && email.value.indexOf("@") !== -1 && email.value.indexOf(".") !== -1 && email.value.lastIndexOf("@") < email.value.lastIndexOf(".") && email.value.lastIndexOf(".") < (email.value.length - 2) 
+        }
+        function validName(name) {
+            return name.value !== "";
+        }
+        function validPhoneNumber(phoneNumber) {
+            return phoneNumber.value !== "" && phoneNumber.value.length < 12;
+        }
+        // If the name, email and contact number valid execute this code
+        /*==============================================================*/
+        if (validName(name) && validPhoneNumber(phoneNumber) && validEmail(email)) {
+                    alert('Thank you for register with us');
+        // If the name, email and contact number are not valid execute this code
+        /*====================================================================*/
+        }else if (!validName(name) && !validPhoneNumber(phoneNumber) && !validEmail(email)) {
```

Comment:
all those **else ifs** are unnecessary and complicate the logic. You don't need to check all of three validations every time, the logic will be simpler if you do it one by one. Also instead of using style.background - create a CSS class called **.error** and add/remove it depending on the logic. If you do that, then if you decide to change how an error style looks like, you just use the CSS in one place.

So the logic should look something like.

```javascript
if(!validName) {
  // apply error style
} else {
 // remove error style
}

if(!validEmail) {
  // apply error style
} else {
 // remove error style
}
//etc..
```

This code can be much simpler than this